cms/templates/js/license-selector.underscore

```diff
@@ -0,0 +1,106 @@
+<div class="license-wrapper">
+    <label class="label setting-label" for="list-license-types">
+        <%= gettext("License Type") %>
+    </label>
+    <ul class="license-types" id="list-license-types">
+        <% var link_start_tpl = '<a href="{url}" target="_blank">'; %>
+        <% _.each(licenseInfo, function(license, licenseType) { %>
+            <li data-license="<%- licenseType %>">
+                <button name="license-<%- licenseType %>"
+                        class="action license-button <% if(model.type === licenseType) { print("is-selected"); } %>"
+                        <% if (license.tooltip) { %>data-tooltip="<%- license.tooltip %>"<% } %>>
+                    <%- license.name %>
+                </button>
+                <p class="tip">
+                    <% if(license.url) { %>
+                        <a href="<%- license.url %>" target="_blank">
+                            <%= gettext("Learn more about {license_name}")
+                                        .replace("{license_name}", license.name)
+                            %>
+                        </a>
+                    <% } else { %>
+                        &nbsp;
+                    <% } %>
+                </p>
+            </li>
+        <% }) %>
+    </ul>
+
+<% var license = licenseInfo[model.type]; %>
+<% if(license && !_.isEmpty(license.options)) { %>
+    <div class="wrapper-license-options">
+        <label class="label setting-label" for="list-license-options">
```

Comment:
```<label>``` elements are only allowed for use with form inputs.  This references a non-widget ```div``` element.  I think a Heading element is more appropriate here.  1 more than its parent (```<h3>``` -> ```<h4>```)